code/modules/power/singularity/narsie.dm

```diff
@@ -113,10 +133,28 @@
 			summon_objective.summoned = FALSE
 			summon_objective.killed = TRUE
 
-	if (GLOB.cult_narsie == src)
-		GLOB.cult_narsie = null
+	send_to_playing_players(span_narsie(span_bold("\"[pick("Nooooo...", "Not die. How-", "Die. Mort-", "Sas tyen re-")]\"")))
```

Comment:
You don't need to tostring the pick.

Reply:
this was already like this i just got rid of the span bold